Styles/Components/input.css

```diff
@@ -0,0 +1,62 @@
+.input {
+  width: 70%;
+  margin: 0.5rem;
+}
+
+.input:first-child {
+  margin-top: 0;
+}
+
+.input input {
+  border: transparent 2px solid;
+  border-radius: 8px;
+  width: 100%;
+  padding: 12px;
+  font-size: 1.2rem;
+  margin: 0;
+}
+
+label {
+  display: block;
+  margin: 0.5rem 0;
+  font-size: 1.2rem;
+}
+
+.input .input-success {
+  border-color: var(--green-3);
+  background: var(--green-6);
+}
+
+.input .input-error {
+  border-color: var(--red-3);
+  background: var(--red-6);
+}
+
+.input .input-success:focus,
+.input .input-error:focus {
+  outline: none;
```

Comment:
Making outline none for an input field is considered as bad practice when Accessibility is considered.

Reply:
[Check here](https://stackoverflow.com/questions/19455499/why-is-outlinenone-not-considered-good-practice)

Reply:
Thanks! I didn't know this, removed outline:none.